docs/deploy/aws.md

```diff
@@ -36,12 +44,36 @@ If you already have a database, take note of the connection string and skip to t
 ## Installation Options
 
 ### On EC2
+#### With DynamoDB
 1. Save the following configuration file as `config.yaml`:
 
    ```yaml
    ---
    database:
-     connection_string: "[DATABASE_CONNECTION_STRING]"
+     type: "dynamodb"
+     dynamodb: 
+       table_name: "[DYNAMODB_TABLE_NAME]"
+       aws_region: "[DYNAMODB_AWS_REGION]"
+       aws_access_key_id: "[AWS_ACCESS_KEY_ID]"
+       aws_secret_access_key: "[AWS_SECRET_ACCESS_KEY]"
```

Comment:
We don't need these 3, just like you don't provide them for the S3 blockstore.

Reply:
Removed